developer/frameworks/articles/data-engine/02-data-collection-api/01-data-record-collection-operations.markdown

```diff
@@ -152,6 +152,14 @@ To delete a `DEDataRecordCollection`, pass a `DEDataRecordCollectionDeleteReques
 Required parameter: `long deDataRecordCollectionId`
 Optional parameters: none
 
+## Deleting a Data Record
+
+To delete a `DEDataRecordCollection`, pass a `DEDataRecordCollectionDeleteRequest` to
```

Comment:
"To delete a **DEDataRecord**, pass a **DEDataRecordCollectionDeleteRecordRequest** to ..